spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncher.java

```diff
@@ -59,26 +62,35 @@ public ModuleLauncher(ModuleResolver moduleResolver) {
 	/**
 	 * Launches one or more modules, with the corresponding arguments, if any.
 	 *
+	 * Modules must be passed in "natural" left to right order, and will be deployed in reverse order
```

Comment:
Also, looking at the launcher in its current state, it does *not* do anything in reverse order (but that's ok, at least for now). Should just keep this comment focused on the contract of what's expected.